hcaptcha.js

```diff
@@ -108,7 +109,7 @@ const getAnswersTF = async (request_image, tasks) => {
 
                 if (
                     data !== undefined &&
-                    data.class.toUpperCase() === request_image.toUpperCase() &&
+                    unhomoglyph(data.class.toUpperCase().trim()) === unhomoglyph(request_image.toUpperCase().trim()) &&
```

Comment:
I'm a little unfamiliar with the unhomoglyph package so just have a few questions. If the text is just plain text (ex: if the text is just "motorobus") will it have any issues evaluating here or anywhere else it is used?

Reply:
Once I'm more acquainted with the package I will approve the request to you can merge and then I will update the readme and such so that I can publish the new package.

Reply:
> Once I'm more acquainted with the package I will approve the request to you can merge and then I will update the readme and such so that I can publish the new package.

What do you wanna know? You can find more info here: https://github.com/nodeca/unhomoglyph